### DISCUSSION TITLE
Wait what? 😆 🤣

### DISCUSSION THREAD

**Discussion by Its3rr0rsWRLD** [openai/whisper#1814] [Show and tell]
**large-v3**

`1
00:00:00,000 --> 00:00:09,960
Images that have notified everyone who left the group in July at 06 hours in the morning and realize that June also marked the beginning of Spring Ultra schedule.`

https://github.com/openai/whisper/assets/92172873/8c4a9cd5-bff0-45a0-8a7d-b6b2d5865616

🤣

**Comment by jongwook** [comment]
real GPT-1 vibe

**Reply by Its3rr0rsWRLD** [reply]
Fr

**Comment by ryanheise** [comment]
I unfortunately couldn't manage to get anything so elaborate using your file, just "Thank you."

@Its3rr0rsWRLD If you manage to be able to reproduce this in particular with the `--word_timestamps True` option enabled, would you be able to share the `.json` output file? I'd like to test a theory that might help to improve the detection and filtering of hallucinations.

**Reply by Its3rr0rsWRLD** [reply]
Hello!

For whisper, I used a program called [WhisperGUI](https://grisk.itch.io/whisper-gui) because I was just using whisper for fun and did not want to set it up fully and deal with python, as python just doesn't like my pc. WhisperGUI, from what I have seen, does not have a repository.

I am curious though, so I will set up whisper in a more controlled and monitored environment and test it there.

Thank you for taking the time to share your thoughts and suggestions.

3rr0r.

Edit:

I also forgot that the source code is all in the folder. 🤦‍♂️

I want to apologize for my indecisiveness and lack of assistance, I have the flu right now and I woke up to a GitHub notification lol.

```
Model: Large-v2
OL: English
Device: 4070 Ti
```

I'll just give the whole program. https://qiwi.gg/file/7HMW1695-WhisperGUI

config_user.json
```
[
  [
    "name",
    "user"
  ],
  [
    "modelBox",
    9
  ],
  [
    "translateBox",
    21
  ],
  [
    "deviceList",
    0
  ],
  [
    "translateToEng",
    0
  ],
  [
    "btnTxt",
    0
  ],
  [
    "btnVtt",
    0
  ],
  [
    "btnSrt",
    1
  ]
]
```

**Reply by ryanheise** [reply]
Ah, I didn't realise it was a different program, so even if I were to fix something in whisper, it would not fix anything in WhisperGUI since it's a different program.

Anyway, the output JSON I was talking about is obtained by running `whisper --model large-v2 --word_timestamps True file.mp3`
